app/server/appsmith-server/src/main/java/com/appsmith/server/migrations/DatabaseChangelog.java

```diff
@@ -827,4 +831,88 @@ public void setExampleAppsInConfig(MongoTemplate mongoTemplate) {
         );
     }
 
+    @ChangeSet(order = "024", id = "clean-action-policies", author = "")
+    public void cleanActionPolicies(MongoTemplate mongoTemplate) {
```

Comment:
Can we rename the function name to correctly represent the functionality? Something along the following lines?
```suggestion
    public void updateErroenousActionIdInPage(MongoTemplate mongoTemplate) {
```